bosh_troubleshooting.prolific

```diff
@@ -31,18 +31,19 @@ Ops files are a way for operators to make edits to a deployment manifest in a re
   - **Append** to a list
 
 ### How?
+1. Get your current bosh manifest by running `bosh -d cf manifest > /tmp/cf.yml`
```

Comment:
I don't have the context on the network policy changes. Should those be done through the original manifest? I guess I am curious why the platform is changing outside of the manifest. It seems like it ought to only change via the manifests.  🤔

Reply:
Check out this story: https://www.pivotaltracker.com/n/projects/2125981 (or this [source](https://github.com/pivotal/cf-onboarding/blob/f462c90ead189d8be5cc34fc988b562960ed18dd/networking_monitoring.prolific#L64)). That's where I saw how to get the latest manifest. 

I think there might have been a few opsfiles that we were forgetting to apply when we got to this step. So another solution would be to list out all the opsfiles that have been used at this point. But I noticed the network policies story grabs the latest manifest and keeps building on that (instead of re-applying all the opsfiles) so I figured that was another viable solution.

Reply:
Hmm, those networking stores were added relatively more recently. It's possible the author didn't keep the whole arc on mind when they were added. I suppose, if it works, it's an improvement :P